### ISSUE TITLE
concat months and years

### ISSUE THREAD

**Issue by stereonom** [soyuka/dpicker#37]
Hi.

Is it possible to add a option to concat `months` and `years`?

Here's why:
In my picker setup I have two instances of `dpicker` for picking a timerange.
There is a `max` for both pickers. You cannot pick any further than `6 months` in advance.
So in the `months` select field there are a maximum of 6 entries.
Adding a `year` select field would add extra complexity for the user as I want the picker to be as simple to use as possible.

I imagine the concat field to display something like `October 2017`, `November 2017`, `December 2017`, `January 2018` or `October '17`, `November '17`, `December '17`, `January '18`

**Comment by soyuka** [comment]
Hmm I'm not really fond of adding such option:
1. it's a really specific use case
2. concatenation is hard to manage because we work with arbitrary strings (a mess to parse)
3. Would change the behavior of how year/months are managed

My suggestion instead would be to have your own small month module! I can help you with it but should be fairly easy to do!
It'd be based on [the renderMonths](https://github.com/soyuka/dpicker/blob/master/src/dpicker.js#L383) original code but by changing some conditions. Also [the following example](https://soyuka.github.io/dpicker/#/?id=previousnext-month-buttons) can be taken as basis.

**Comment by stereonom** [comment]
1. yeah, that's right
2. It's shouldn't be that hard to parse, because the value of the `<option>` could be definied in a way it is easy to parse. only the text inside the `<option>` is a mess. e.g. `<option value="2017-08">August '17</option>`
3. Yeah, I give u that.

I'll try building a module tomorrow. I come back to you if I need help.

**Comment by stereonom** [comment]
#40
